src/feditest/nodedrivers/imp/__init__.py

```diff
@@ -15,20 +15,24 @@
 from feditest.reporting import trace
 
 
+_HEADERS = {
+    "User-Agent": "feditest/0.0.0"
+}
```

Comment:
Could the 0.0.0 be pulled from the package version?

Reply:
Yes, I think so. I'll modify it to do that.